src/ol/PluggableMap.js

```diff
@@ -663,7 +670,9 @@ class PluggableMap extends BaseObject {
    */
   getEventPixel(event) {
     const viewportPosition = this.viewport_.getBoundingClientRect();
-    const eventPosition = event.changedTouches ? event.changedTouches[0] : event;
+    const eventPosition = event instanceof TouchEvent ? event.changedTouches[0] :
```

Comment:
`TouchEvent` is not known on all browsers. I'd prefer a cast of `event` to `TouchEvent` here:
```js
 const eventPosition = 'changedTouches' in event ? /** @type {TouchEvent} */ (event).changedTouches[0] :`
```